docs/source/start-routingspec.rst

```diff
@@ -662,22 +717,200 @@ so consider the name before saving.
 
 Once the new window comes up the user should address the following fields, at least.
 
-+-------------------------+---------------------------------------------------+
-|**Field**                |**Description**                                    |
-+=========================+===================================================+
-|Data Source              |Location in Database                               |
-+-------------------------+---------------------------------------------------+
-|Destination              |consumer (pipe, database, file, etc)               |
-+-------------------------+---------------------------------------------------+
-|Output Format            |set output format                                  |
-+-------------------------+---------------------------------------------------+
-|Date/Time - Since/Util   |define time window for message retreival           |
-+-------------------------+---------------------------------------------------+
-|Platform Selection       |Specify a platform or network list                 |
-+-------------------------+---------------------------------------------------+
++-------------------------+------------------------------------------------+
+|**Field**                |**Description**                                 |
++=========================+================================================+
+|Data Source              |Location in Database                            |
++-------------------------+------------------------------------------------+
+|Destination              |consumer (pipe, database, file, etc)            |
++-------------------------+------------------------------------------------+
+|Output Format            |set output format                               |
++-------------------------+------------------------------------------------+
+|Date/Time - Since/Util   |define time window for message retrieval        |
++-------------------------+------------------------------------------------+
+|Platform Selection       |Specify a platform or network list              |
++-------------------------+------------------------------------------------+
+
+The **Data Source** will populate with a drop down menu consisting of the
+ source or sources added in step 3.  The **Destination** will populate with 
+ some options including but not limited to the database, file and pipe.  
+ To set up a routing spec to go into the database, select the applicable 
+ database.  The **Date/Time** format is only applicable for sources
+ that are not static.  For example, sources of type lrgs or abstract web
```

Comment:
suggestion: 
 The **Date/Time** selection is used with Data Sources
 that require selecting a range of Date/Time.

Date/Time is ignored by 'static' Data Sources, that provide a pre-defined time window; such as always providing the last twenty four hours of data.